includes/admin/tools.php

```diff
@@ -1467,6 +1468,26 @@ function edd_tools_sysinfo_get() {
 	return $return;
 }
 
+/**
+ * Get TLS Info
+ *
+ * @since       2.9
+ * @return      string
+ */
+function edd_tools_tlsinfo() {
+
+	if( ! function_exists( 'curl_init' ) ) {
+		return 'TLS info unavaible while cURL is disabled';
```

Comment:
Noting a spelling mistake on "unavaible", and also this should be wrapped in a transalation function.

Reply:
Yikes. Sorry about that.

Reply:
All good!